fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourceManagerHelper.java

```diff
@@ -52,4 +54,27 @@ default void addReloadListener(IdentifiableResourceReloadListener listener) {
 	static ResourceManagerHelper get(ResourceType type) {
 		return ResourceManagerHelperImpl.get(type);
 	}
+
+	/**
+	 * Registers a built-in resource pack.
+	 *
+	 * <p>A built-in resource pack is an extra resource pack provided by your mod which is not always active, it's similar to the "Programmer Art" resource pack.
+	 *
+	 * <p>Why and when to use it? A built-in resource pack should be used to provide extra assets/data that should be optional with your mod but still directly provided by it.
+	 * For example it could provide textures of your mod in another resolution, or could allow to provide different styles of your assets.
+	 *
+	 * <p>The {@code subPath} corresponds to a path in the JAR file which points to the resource pack folder. For example the subPath can be {@code "resourcepacks/extra"}.
```

Comment:
How does "path in the JAR" relate to the file view I as a developer have over my project structure? Is this relative to `src/` ? `src/main/resources/`? `src/main/resources/assets/`?
A minimal usage example would be hugely beneficial here.

Reply:
"Path in the JAR" is already very explicit to me, it's a path relative to the JAR root.

Anything put in `src/main/resources` is put in the JAR root.

Reply:
I didn't know that, but okay.